### ISSUE TITLE
Can't change guake bgcolor

### ISSUE THREAD

**Issue by eblis** [Guake/guake#1359]
- **Guake Version**: 3.0.5
  - Linux Mint 19

I'm trying to set the guake background color when connected to remote SSH instances by running `guake --bgcolor=color` but i get the following error:

```
Traceback (most recent call last):
  File "/usr/bin/guake", line 10, in <module>
    sys.exit(exec_main())
  File "/usr/lib/python3/dist-packages/guake/main.py", line 356, in exec_main
    if not main():
  File "/usr/lib/python3/dist-packages/guake/main.py", line 308, in main
    remote_object.set_bgcolor(options.bgcolor)
  File "/usr/lib/python3/dist-packages/dbus/proxies.py", line 70, in __call__
    return self._proxy_method(*args, **keywords)
  File "/usr/lib/python3/dist-packages/dbus/proxies.py", line 145, in __call__
    **keywords)
  File "/usr/lib/python3/dist-packages/dbus/connection.py", line 651, in call_blocking
    message, timeout)
dbus.exceptions.DBusException: org.freedesktop.DBus.Python.AttributeError: Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/dbus/service.py", line 707, in _message_cb
    retval = candidate_method(self, *args, **keywords)
  File "/usr/lib/python3/dist-packages/guake/dbusiface.py", line 95, in set_bgcolor
    self.guake.set_bgcolor(bgcolor)
AttributeError: 'Guake' object has no attribute 'set_bgcolor'

```

**Comment by Speedkore** [comment]
i want the same because many times i write commands on servers thinking that i'm in local, and one day i will fuck up a server, i really want to put the background in red only for selected tab! I'm in version 3.2.2 and still doesn't work, and btw many thanks for Guake i use it every day!

**Comment by gsemet** [comment]
I’ll look at it ASAP !

**Comment by varelajp** [comment]
Please...
Independent tabs theme.
It would be great!

Kind regards...

**Comment by aichingm** [comment]
This should be fixed by #1378

**Comment by gsemet** [comment]
The --bgcolor param has been fixed.
